packages/amplify-cli/src/utils/post-install-initialization.ts

```diff
@@ -8,7 +8,6 @@ import { print } from '../context-extensions';
  * It copies some assets needed by dependency packages to the .amplify folder
  */
 export const postInstallInitialization = async () => {
-  print.info(`Initializing new Amplify CLI version...`);
```

Comment:
Since `print` is no longer used, please remove the import

Reply:
Done!